pkg/scheduler/api/resource_info.go

```diff
@@ -296,6 +296,16 @@ func (r *Resource) Multi(ratio float64) *Resource {
 	return r
 }
 
+// ScaleResourcesWithRatios multiples the resource with ratio provided for CPU, Memory, and ScalarResources
+func (r *Resource) ScaleResourcesWithRatios(cpuRatio float64, memRatio float64, scalarRatio float64) *Resource {
+	r.MilliCPU *= cpuRatio
+	r.Memory *= memRatio
+	for rName, rQuant := range r.ScalarResources {
```

Comment:
What if different scalar has its own ratio? And scalar resource support overcommit?

Reply:
If super-resolution is not supported, we can set it to 1 through the configuration file. Users can freely use the configuration file to set the required value

Reply:
This is also an improvement compared to the previous

Reply:
fixed